core/src/main/java/de/danoeh/antennapod/core/util/DBWriterForFeedPreferencesUtil.java

```diff
@@ -0,0 +1,10 @@
+package de.danoeh.antennapod.core.util;
+
+import de.danoeh.antennapod.core.feed.Feed;
+import de.danoeh.antennapod.core.storage.DBWriter;
+
+public class DBWriterForFeedPreferencesUtil {
+    public static void saveFeedPreferences(Feed feed) {
+        DBWriter.setFeedPreferences(feed.getPreferences());
```

Comment:
Why don't just call the method directly? I don't think it is necessary to wrap this in another utility class.

Reply:
Two reasons, one - only FeedSettingsPreferenceFragment is calling saveFeedPreferences (UI->DB), second - i created it with thought about other DB calls with similar purpose to keep business logic in future in one place.

I don't really understand what is Feed, FeedItem and FeedMedia. 
FeedItem is one of elements of Feed but FeedItem is actually holding Feed property. 
FeedMedia is a playable media in FeedItem but FeedMedia is holding a FeedItem property as well as FeedMedia holding property FeedItem.
It may seem like an out of context question but it its related logic in FeedSettingsPreferenceFragment and FeedPreference.

Reply:
> i created it with thought about other DB calls with similar purpose to keep business logic in future in one place.

Actually, DBWriter already is the abstraction for the database operations.

> I don't really understand what is Feed, FeedItem and FeedMedia.

Feeds ("subscriptions") can have items. Each item can theoretically have any number of media files. Though, we only use a single one currently (or none for text-only items).

> FeedItem is one of elements of Feed but FeedItem is actually holding Feed property.

Yeah, the 3 classes are coupled pretty strongly. Getting the corresponding feed for a FeedItem is needed in many places like when deciding if it should auto-skip. I think we can leave that as it is for now.